src/shims/foreign_items.rs

```diff
@@ -102,6 +118,78 @@ pub trait EvalContextExt<'mir, 'tcx: 'mir>: crate::MiriEvalContextExt<'mir, 'tcx
         }
     }
 
+    /// Lookup the body of a function that has `link_name` as the symbol name.
+    fn lookup_exported_symbol(
+        &mut self,
+        abi: Abi,
+        link_name: Symbol,
+    ) -> InterpResult<'tcx, Option<&'mir mir::Body<'tcx>>> {
+        let this = self.eval_context_mut();
+        let tcx = this.tcx.tcx;
+        let load_mir = |this: &crate::MiriEvalContext<'_, 'tcx>, instance: &ty::Instance<'tcx>| {
+            check_abi(abi, instance.ty(tcx, ty::ParamEnv::reveal_all()).fn_sig(tcx).abi())?;
```

Comment:
I assume this can be removed once https://github.com/rust-lang/rust/pull/85087/files lands?

Reply:
Yes.
(I have tested it with this line [removed](https://github.com/rust-lang/miri/commit/5752059f1b9b2bcc9946f4fa5112c81f8bb49527) in rust-lang/rust#85087.)

Reply:
Great. :)
The `load_mir` should then probably also be inlined.